Makefile

```diff
@@ -323,6 +323,7 @@ build/chart/index.yaml.$(YEAR_MONTH_DAY): build/chart/index.yaml
 build/chart/: build/chart/index.yaml build/chart/index.yaml.$(YEAR_MONTH_DAY)
 
 install-large-chart: build/toolchain/bin/helm$(EXE_EXTENSION) install/helm/open-match/secrets/
+	$(KUBECTL) apply -f install/open-match-metadata-server.yaml
```

Comment:
This is repetitive. Make this a build dependency. You also are missing the `kubectl$(EXE_EXTENSION)` dependency to call this anyway.